### PR TITLE
Add support for moving itertools imports to six.moves

### PR DESCRIPTION
**Pull request by rowillia**

### CODE REVIEW

**Comment by rowillia** [comment]
@brettcannon @takluyver @daira Ping!  Anything I can do here for this change?

**Comment by takluyver** [comment]
Looks reasonable enough. Does it handle `from itertools import imap as foo`? I see some code in the fixer that looks like it might. If so, a test would be good.

What about `import itertools; itertools.imap(...)`? I forget if other similar fixes handle things like that.

**Comment by rowillia** [comment]
Thanks for taking a look @takluyver!  You were right, this wasn't properly handling just module imports.  Fixed by pulling in `fix_itertools` as well and modifying it to support `six`

File: libmodernize/fixes/fix_itertools_six.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+""" Fixer for itertools.(imap|ifilter|izip) --> (map|filter|zip) and
+    itertools.ifilterfalse --> itertools.filterfalse (bugs 2360-2363)
+    imports from itertools are fixed in fix_itertools_import.py
+    If itertools is imported as something else (ie: import itertools as it;
+    it.izip(spam, eggs)) method calls will not get fixed.
```
Comment (takluyver) [inline libmodernize/fixes/fix_itertools_six.py]: I think this docstring should mention six. Otherwise, this looks good to me.

Comment (rowillia) [inline libmodernize/fixes/fix_itertools_six.py]: Done

**Comment by rowillia** [comment]
@takluyver Done, Thanks!

**Pull request merged**